src/components/TradeWidget/index.tsx

```diff
@@ -26,20 +30,50 @@ const SubmitButton = styled.button`
 
 const TradeWidget: React.FC = () => {
   const { networkId } = useWalletConnection()
-  const DAI = _getTokenDetails('DAI', networkId)
-  const USDC = _getTokenDetails('USDC', networkId)
+  const fallBackNetworkId = networkId ? networkId : Network.Mainnet // fallback to mainnet
```

Comment:
should we fallback to Mainnet? Traditionally it's been local

Reply:
My idea is to get the mainnet token addresses because for those we are more likely to have all the tokens with the proper images listed. 

Also, we currently have only Mainnet and Rinkeby networks https://github.com/gnosis/dex-react/blob/develop/src/api/index.ts#L29, no local so far.

Maybe I'm confusing the concepts of connecting to a local network vs loading token addresses from different networks. If connected to local, the node could be connected to either Mainnet or Rinkeby.
In that case, we have two sets of options:
1. Connect to a local node OR a web3provider such as metamask
2. Pick the network we'll operate on, such as Mainnet or Rinkeby

This fallback only addresses `2`.

Does that make sense?

Reply:
Hi! 

I understand this faultback is only relevant while the wallet is not connected and we don't know the network yet.

In this case options are:
- *Do not show anything*: I think it's undesirable in this case, cause we want to allow the user to play with the trade page although he is not connected
- *Show a default network*: This is what I understand Leandro did and I agree that mainnet can be a good default

However, I think we can do something else. It can be address in a different PR (if you agree we can add it to our backlog). 
The app will be deployed in a server, the URL for rinkeby will be different as the URL for mainnet, i.e:
* `batch.trade`
* `rinkeby.batch.trade`
I think we should know if we are targeting mainnet or rinkeby, and use it as our default. One way to do this is setup a constant on app initialization using the url of the app:
* `batch.trade` --> mainnet
* `rinkeby.batch.trade` ---> rinkeby
* null --> in this case, we can decide in the component what to do. 

WDYT?

Reply:
I like the idea of the constant in the initialization, given the two different deployments. In that case though, the logic for detecting which is the default network should not be in the component, but instead in whatever provides the `networkId`, and this value should never be `undefined`.

Other point somewhat related: what should happen in case the user is using our Rinkeby instance and (using Metamask for example) switches the network to Mainnet? Should that be allowed? How would that look like from a legal stand point?